### PR TITLE
added fake_news_tracker app proposal

### PR DESCRIPTION
**Pull request by jmartinhoj**
Added a proposal of an app to track fake news about the new coronavirus!

### CODE REVIEW

File: software/proposals/fake_news_tracker.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+### What it is
+
+### What it is not
+
+### Design
+
+### Data/Experiments
+
+## 💪 Impact of Potential Solution
```
Comment (2001kraft) [inline software/proposals/fake_news_tracker.md:38]: Please state the impact

File: software/proposals/fake_news_tracker.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+## 🙌 Criteria for Success
```
Comment (2001kraft) [inline software/proposals/fake_news_tracker.md:40]: Please state KPIs

File: software/proposals/fake_news_tracker.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+## 😅 Known Issues
```
Comment (2001kraft) [inline software/proposals/fake_news_tracker.md:42]: What are the known issues with this project?

File: software/proposals/fake_news_tracker.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+With more and more people infected by the coronavirus, more and more misinformation and fake news are emerging. We no longer can distinguish between what is true and what is fake.
+
+### 👨‍🔬Hypothesis
+
+Simple mobile app that shows various news topics that are spreading and tells if they are true or fake. It is important for people to know exactly what is going on in their country and the world.
+
+### 🤔 Assumptions
+
+## 💻Implementation Suggestion
```
Comment (2001kraft) [inline software/proposals/fake_news_tracker.md:26]: Please fill out the implementation—this is crucial for mobile products

**Review by 2001kraft — CHANGES_REQUESTED** [review]
Please fill out the template completely.

**Comment by mazaira** [comment]
Closing by lack of feedback. Please feel free to re-open if there would be any changes from the current state.

**Pull request closed without merging**